src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -103,6 +106,17 @@ public Person findPersonByName(Name name) throws PersonNotFoundException {
             throw new PersonNotFoundException();
         }
     }
+    //@@author steven-jia
+    @Override
+    public void findPersonInTransaction(Name name) throws PersonNotFoundException {
+        Set<Person> matchingPersons = addressBook.getPersonList()
+                .stream()
+                .filter(person -> person.getName().equals(name))
```

Comment:
I think you should use Person for argument instead of name, and do "person.equals(person)". It is more accurate.